Tests/test_imageops.py

```diff
@@ -94,6 +94,22 @@ def test_scale(self):
         newimg = ImageOps.scale(i, 0.5)
         self.assertEqual(newimg.size, (25, 25))
 
+    def test_colorize(self):
+        # Test the colorizing function
+
+        # Grab test image
+        i = hopper("L").resize((15, 16))
```

Comment:
Let's use the more standard `im` variable name. We can update the other test too.

Reply:
ok great, sounds good, the `i` was confusing to me too